Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ignotus666 commented on _translator-files/po/fr/Tips-Tricks-More.po:
```diff
@@ -82,8 +83,23 @@ msgstr "### Découvrir les répétitions de groupes à distance"
+"couvre des sujets tels que la configuration matérielle et logicielle, y "
+"compris des exemples et des conseils pour les nouveaux venus dans le "
+"domaine. Il inclut également une section sur Jamulus en comparaison avec "
+"d'autres solutions."
```
I wonder why Weblate decided to wrap all the lines in this doc (I checked that the add-on preventing this is installed)?

pljones replied:
It does make reviewing on Github nigh on impossible...

ignotus666 replied:
Do you mean you find it harder when it's wrapped or when it's not?

pljones replied:
I mean I find it hard because it has been wrapped.  Seeing what exactly changed isn't very easy due to the wrapping obscuring everything.

So long as the wrapping is stable between updates, I don't mind one way or the other -- but I don't want it changing it ad hoc.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/installation-for-windows/
Translation: Jamulus/Installation-for-Windows

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Directories
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/

ann0see commented on _translator-files/po/es/FAQ.po:
```diff
@@ -6,15 +6,15 @@
-"PO-Revision-Date: 2022-08-15 21:26+0000\n"
-"Last-Translator: ignotus <please@seeGitHubOrWeblate.local>\n"
+"PO-Revision-Date: 2022-09-28 19:55+0000\n"
+"Last-Translator: ignotus <darylo1@hotmail.com>\n"
```
I've now moved to creating fake E-Mail accounts...

ann0see reviewed (commented):
@ignotus666 maybe you need to enable the wrap plugin in multiple components?
